app/services/ckan/v26/inspire_updater.rb

```diff
@@ -0,0 +1,31 @@
+module CKAN
+  module V26
+    class InspireUpdater
+      def call(dataset, package)
```

Comment:
Perhaps we could provide just the id of the dataset instead of the whole object?

Reply:
I was going more for uniformity and avoiding the import worker knowing too much about what the individual updaters need. It's possible the inspire updater would need to see the dataset attributes.